### PR TITLE
Make text-orientation:sideways-right an alias for sideways in Gecko

### LINKED ISSUE #15214: text-orientation should accept sideways-right as an alias of sideways
Spec: https://drafts.csswg.org/css-writing-modes-3/#valdef-text-orientation-sideways-right

### PR DESCRIPTION
**Pull request by mbrubeck**
Fixes #15214. r? @upsuper

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15268)

### CODE REVIEW

File: components/style/properties/data.py [hunk 1]
```diff
@@ -48,6 +49,7 @@ def __init__(self, name, values, gecko_constant_prefix=None,
         self.gecko_enum_prefix = gecko_enum_prefix
         self.extra_gecko_values = (extra_gecko_values or "").split()
         self.extra_servo_values = (extra_servo_values or "").split()
+        self.gecko_alias_values = {} if gecko_alias_values is None else gecko_alias_values
```
Comment (jdm) [inline components/style/properties/data.py]: This is equivalent to `gecko_alias_values or {}`.

Comment (mbrubeck) [inline components/style/properties/data.py]: Fixed.

File: components/style/properties/properties.mako.rs [hunk 2]
```diff
@@ -1676,6 +1675,7 @@ pub fn get_writing_mode(inheritedbox_style: &style_structs::InheritedBox) -> Wri
         computed_values::text_orientation::T::mixed => {},
         computed_values::text_orientation::T::upright => {},
     % endif
+        computed_values::text_orientation::T::sideways_right => {},
```
Comment (upsuper) [inline components/style/properties/properties.mako.rs]: If `sideways-right` is an alias of `sideways`, why do they have different effect here?

Gecko drops `sideways-right` value (parses it as `sideways`), which I guess Servo can do as well. Is there any reason to keep Servo's `sideways-right` separate from `sideways`?

**Comment by mbrubeck** [comment]
Servo layout does not actually support `text-orientation` at all, so I updated this patch to make it a Gecko-only property for now.

**Review by upsuper — APPROVED** [review]
LGTM

**Comment by Wafflespeanut** [comment]
The unit tests seem to fail.

**Comment by mbrubeck** [comment]
@bors-servo r=upsuper

* Pushed a fix for the unit tests.

**Pull request merged**